package.json

```diff
@@ -7,6 +7,10 @@
   "author": "Bryan Helmig <bryan@zapier.com>",
   "license": "UNLICENSED",
   "main": "schema.js",
+  "files": [
+      "/exported-schema.json",
+      "/lib/**/*.js"
```

Comment:
needs:

* schema.js

also in `lib/utils`, 3/5 of those files are just for docs. If we're really trying to cut down on size, we'd either explicitly exclude those 3 or include the two (`makeSchema` and `makeValidator`) that we need

Reply:
Fixed in 58a8cb6.

I tested on my local. We actually need `lib/utils/exportSchema.js` and `lib/utils/links.js` because they're imported internally somewhere in `zapier-platform-schema`. By this rationale, I think it's better to include all the files in `/lib` as other modules in `zapier-platform-schema` may want to import them.